lib/stack_master/template_compilers/sparkle_formation.rb

```diff
@@ -22,7 +22,7 @@ def self.compile(template_dir, template, compile_time_parameters, compiler_optio
         sparkle_template.compile_state = create_state(definitions, compile_time_parameters)
       end
 
-      JSON.pretty_generate(sparkle_template)
+      JSON.pretty_generate(sparkle_template.dump)
```

Comment:
I noticed the compiled SparkleFormation template wasn't always pretty formatted during Cucumber tests.

We avoid this problem by dumping the template to a hash before converting it to a JSON string.